openshift/templates/api-config.json

```diff
@@ -0,0 +1,239 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "annotations": {
+      "description": "Configmap template for the api service. This template may not include real value of secrets; you need to manually replace the value in Openshift.",
+      "tags": "${NAME}-${TAG_NAME}"
+    },
+    "name": "${NAME}-config"
+  },
+  "objects": [
+    {
+      "kind": "ConfigMap",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${TAG_NAME}-config",
+        "labels": {
+          "app": "${NAME}-${TAG_NAME}",
+          "app-group": "${APP_GROUP}",
+          "template": "${NAME}-config"
+        }
+      },
+      "data": {
+        "POD_TESTING": "true",
+        "SENTRY_DSN": "https://<account>@sentry.io/<project>",
```

Comment:
@pwei1018 I'm guessing we don't need many of these variables since this isn't an auth application anymore. it's a search?

Reply:
Yes, most of variables you may not need it.